proposals/nnnn-AOA-multiplexing.md

```diff
@@ -9,45 +9,64 @@
 ## Introduction
 
 Add "multiplexing" to AOA transport. ![Overview](../assets/proposals/nnnn-AOA-multiplexing/nnnn-AOA-multiplexing_overview.png)
-This change introduces a better user experience as followings:
-- Multi navigation apps to be able to exchanged by user.
-- None navigation apps can select not only BT/SPP but AOA. 
-- In the future, projection will be applied to not only navigation but also other category application then, multi "projection apps" can be supported.
+This change introduces a better user experience as follows:
+- Multiple navigation apps can be selected by user without any operation on HS. It's same as iPhone's UX.
+- Non-navigation apps can use not only BT/SPP but AOA as transport. 
+- In the future, projection will be applied to not only navigation but also other category app then, multi "projection apps" can be supported.
+ NOTICE: It will be depend on HU capability, "Whether multi projection app can be run in parallel (multi view port for each projection app) or not".
 
 ## Motivation
 
-Currently, only one AOA app can connnect to SDL core because of AOA limitation. Before that, our assumption of AOA, only 1 navigation app can connect to SDL core. No more app is assumed. However, several navigation apps will come in the near future. And, projection functionality will be opened for not navi but other category apps which requires wide bandwidth such USB. So, for better and flexible UX, it should be valuable to support multiple app on AOA.
+Currently, only one AOA app can connect to SDL core because of AOA limitation. However, a few navigation apps will be released. And, video projection functionality will be opened for not only navigation but also other category apps. And apps which is not required projection but required wide band-width and stable transport, may be came. This proposal is for supporting and realizing these requirements.
```

Comment:
Can you please clarify the third sentence in this paragraph?

Reply:
Is "third sentence" 
  "And, video projection functionality will be opened for not only navigation but also other category apps."
Or 
  "This proposal is for supporting and realizing these requirements."

If "This proposal is for supporting and realizing these requirements." then, 
"This proposal provide the solution which meets these requirement by supporting multiplexing in AOA", this is revised one. Is this clarified?

If "And, video projection functionality will be opened for not only navigation but also other category apps." then, 
Could you clarify "what's point not clear"?

Reply:
@tsuzuki-uie I am looking for clarification of this sentence " And apps which is not required projection but required wide band-width and stable transport, may be came."

Reply:
"And not projection but using wide band-width and stable transport apps (ex. using high quality audio streaming or pass thru), may be came in the future. This means more than two apps will require USB, in the case of Android handset, this means AOA multi be required.", this is revised one.

Reply:
Can you change "may be came in the future." to "may be considered in the future." ?

Reply:
OK.